src/main/java/com/example/petbuddybackend/dto/notification/NotificationDTO.java

```diff
@@ -0,0 +1,16 @@
+package com.example.petbuddybackend.dto.notification;
+
+import lombok.Builder;
+
+import java.time.ZonedDateTime;
+
+@Builder
+public record NotificationDTO(
+        Long notificationId,
+        Long objectId,
+        String objectType,
+        ZonedDateTime createdAt,
+        String message,
```

Comment:
Z tym messegem to bym się zastanowił. Na frontendzie oni supportują język angielski i polski, więc pytanie czy message nie powinien być na frontendzie handlowany

Reply:
moze tak byc